### DISCUSSION TITLE
Add cost of individual devices to the UI (Energy Dashboard)

### DISCUSSION THREAD

**Discussion by mdehaas** [home-assistant/frontend#10538] [Dashboard]
I would like to be able to quickly view the cost of an individual device. I feel there are different ways to do this:

- The simplest one, add it to the tooltip in the graph when hoovering over the specific bar
- Add extra graph for this, this would allow to split the usage in different tarif tiers which allows an insight to change the period in which you use the device.
- The options above could also be incorporated in the existing graph where the tooltip shows a total, tarif1 and tarif2 for both KW/h and cost. I feel this might clutter up the graph, it would however save on the total length of the dashboard.

There are probably other options but these are the ones that I can think of quickly.

Kind regards,
Marcel de Haas

**Comment by sle118** [comment]
Definitely something i'd be interested in, as it would provide a quick glimpse of each device's impact on budget.  For example, how much is that spa costing during the summer... or during the winter!

**Comment by m-klecka** [comment]
I'd also love to have this feature.

**Comment by modo1302** [comment]
lookig forward to having such an option! :D
